File: src/main/java/com/github/unchama/seichiassist/data/player/StarLevel.kt
package com.github.unchama.seichiassist.data.player

data class StarLevel(val fromBreakAmount: Int = 0, val fromConnectionTime: Int = 0, val fromEventAchievement: Int = 0) {
  fun total() = fromBreakAmount + fromConnectionTime + fromEventAchievement
}

File: src/main/java/com/github/unchama/seichiassist/data/player/StarLevel.scala
package com.github.unchama.seichiassist.data.player

case class StarLevel(fromBreakAmount: Int = 0, fromConnectionTime: Int = 0, fromEventAchievement: Int = 0) {

  def total(): Int = fromBreakAmount + fromConnectionTime + fromEventAchievement

}